doc/en/components/inputs/select.md

```diff
@@ -0,0 +1,164 @@
+mentionedTypes: ['Select']
+---
+
+# $Platform$ Select
+The $ProductName$ Select component allows a single selection from a list of items, placed in a dropdown. This form control offers a quick items list navigation, including selection, based on a single or multiple characters match. 
+
+## $Platform$ Select Example
+
+<code-view style="height:220px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/inputs/select-overview" alt="$Platform$ Select Example"
+           github-src="inputs/select/overview">
+</code-view>
+
+<div class="divider--half"></div>
+
+## Usage
+
+<!-- WebComponents -->
+First, you need to install the $ProductName$ by running the following command:
+
+```cmd
+npm install {PackageWebComponents}
+```
+<!-- end: WebComponents -->
+
+Before using the `Select` component, you need to register it together with its additional components:
+
+```ts
+import { defineComponents,
+         IgcSelectComponent,
+         IgcSelectItemComponent,
+         IgcSelectGroupComponent,
+         IgcSelectHeaderComponent } 
+from 'igniteui-webcomponents';
+
+defineComponents(IgcSelectComponent, IgcSelectItemComponent, IgcSelectGroupComponent, IgcSelectHeaderComponent);
+```
+
+> [!NOTE]
+> Please note that the select header and group components are not mandatory unless you want to use them.
+
+To start using the component add the `igc-select` along with a list of `igc-select-item`'s to choose from:
+
+```html
+<igc-select>
+    <igc-select-item value="Orange">Orange</igc-select-item>
+    <igc-select-item value="Apple">Apple</igc-select-item>
+    <igc-select-item value="Banana">Banana</igc-select-item>
+    <igc-select-item value="Mango">Mango</igc-select-item>
+</igc-select>
+```
+
+### Select 
+
+The `igc-select` component can be used inside and `igc-form` thus it exposes a `Name` property to be registered with. It also has a `Label`, `Placeholder`, and `Size` properties. The `Outlined` property is used for styling purposes only when it comes to the Material theme. Except for the default one, the component provides a few other slots including `header`, `footer`, `helper-text`, `prefix`, `suffix`, and `toggle-icon`. 
+
+### Item
+
+The `igc-select-item` component allows the users to declaratively specify a list of options to be used by the `igc-select` control. Each item provides a `Value` property that represents the data it carries upon selection. The `SelectItem` has a default slot which allows you to specify the content of the item. This content will only be used for visual purposes and will not be used in case the `Value` property is not present on the `igc-select-item`. You could also provide custom content to be rendered before or after the `SelectItem` content using the `prefix` and `suffix` slots. You could predefine a selected item by setting the `Selected` property. You could also disable some or all items via the `Disabled` property.
```

Comment:
```suggestion
The `igc-select-item` component allows the users to declaratively specify a list of options to be used by the `igc-select` control. Each item provides a `Value` property that represents the data it carries upon selection. The `SelectItem` has a default slot which allows you to specify the text content of the item. This text content will be used as value in case the `Value` property is not present on the `igc-select-item`. You could also provide custom content to be rendered before or after the `SelectItem` content using the `prefix` and `suffix` slots. You could predefine a selected item by setting the `Selected` property. You could also disable some or all items via the `Disabled` property.
```